lib/app/shared/helper_functions/helper_functions.dart

```diff
@@ -0,0 +1,12 @@
+String generateDefaultAccountName(
```

Comment:
logic is incorrect.. User can modify name.. so you cannot compare with that name...